### PR TITLE
Changes the color of the "Delete User" and "X" exit buttons on hover.

### LINKED ISSUE #43416: The pop-up's Delete button does not change color when hovered over
## Steps to reproduce:
1. Click on the `user delete` button on a linked workpackage
2. a pop-up will appear
3. hover over the `cancel` button, the color is changed
image
4. But when hover over `Delete **'s account` button then the color is not changed
image

### PR DESCRIPTION
**Pull request by theTyster**
* Resolves: #43416 

## Summary
This is my first pull request for Nextcloud! I hope I have submitted everything correctly. I took a look at the results with Wave and didn't see any issues but was unsure how to include these tests in this pull request. Regardless, everything seemed ok.

This is a screenshot of the initial UI without any hover attributes enabled.
Initial

This is a screenshot of the UI without any hover attributes enabled now. (Notice the "X" no longer has a border unless it is hovered"
02-24-2024_02:53%:03PM

This is a screenshot of the red button on hover.
02-24-2024_02:52%:51PM

The color change is slight. I used the existing `$color-error-hover` variable.
```scss
$color-error-hover: mix($color-error, $color-main-background, 80%) !default;
```
## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Comment by theTyster** [comment]
@emoral435 

Just want to make sure I submitted this right. I can't tell if I submitted it wrong or if it's just taking a while to get to it. No pressure either way.

**Comment by emoral435** [comment]
@theTyster Hi! Thank you so much for the contribution! Big thank you from the team at Nextcloud 💙
You submitted your PR correctly! However, this PR cannot be merged, as from checking, we have the border around the 'X' close button even while not hovering in order to adhere to accessibility standards. The contrast without the border is not 3:1.

image
I will have to close this PR, but again, big thank you from the team over at Nextcloud :)

**Comment by theTyster** [comment]
@emoral435 

> The contrast without the border is not 3:1.

Thanks for pointing that out! I reverted the change I made. It appears I still have some getting used to with WAVE. I appreciate the pointers.

**Comment by emoral435** [comment]
@theTyster Of course, no problem!

**Review by emoral435 — APPROVED** [review]
LGTM! I do not believe that this will affect any of our NcVue libraries that have accessibility in mind, but this could catch any spots that any other legacy apps use in terms of styling, so its a yes from me 👍

File: core/css/inputs.scss [hunk 1]
```diff
@@ -323,6 +323,10 @@ input[type='reset'] {
 		background-color: var(--color-error) !important;
 		border-color: var(--color-error) !important;
 		color: #fff !important;
+		&:hover{
+			background-color: var(--color-error-hover) !important;
+			border-color: var(--color-primary-element) !important;
```
Comment (susnux) [inline core/css/inputs.scss:328]:
```suggestion
			border-color: var(--color-main-text) !important;
```

I think this is more consistent with our current other components

**Review by susnux — APPROVED** [review]

**Comment by susnux** [comment]
@theTyster can you rebase onto current master? Because there are conflicts currently.

**Review by Pytal — APPROVED** [review]

**Comment by theTyster** [comment]
> @theTyster can you rebase onto current master? Because there are conflicts currently.

Started working on that as soon as I saw it was approved. 😄

**Comment by susnux** [comment]
Thank you very much!
And sorry for the delay, somehow this was under the radar.
Looking forward for more contributions :)

**Comment by susnux** [comment]
CI is broken on forks, so I need to merge this first to a different branch

**Comment by theTyster** [comment]
> Thank you very much! And sorry for the delay, somehow this was under the radar. Looking forward for more contributions :)

It is truly an honor. I can't wait to contribute more.

Sorry about my crumby commit messages. I learned about semantic commits after I started contributing to Nextcloud. 😬

**Comment by theTyster** [comment]
> CI is broken on forks, so I need to merge this first to a different branch

Is there something I should have done differently that could have helped?

**Comment by susnux** [comment]
> Is there something I should have done differently that could have helped?

No the CI for cypress simply fails for forks, nothing to worry :)

**Pull request merged**